BeastXUD.md

```diff
@@ -0,0 +1,95 @@
+This guide is written for professional market makers & traders to build an always-on xud node with a powerful mini pc.
+
+![BeastXUD setup (plants are optional)](images/BeastXUD.jpg)
+
+Two options are available:
+
+1. **Light setup** using [Neutrino](https://github.com/lightninglabs/neutrino) and [Infura](https://infura.io/). This keeps the setup light-weight, but requires to trust entities like [Infura](https://infura.io/).
+2. **Full setup** using [bitcoind](https://github.com/bitcoin/bitcoin/), [litecoind](https://github.com/litecoin-project/litecoin) and [geth](https://github.com/ethereum/go-ethereum). Requires more resources and an SSD, but keeps the setup trustless.
+
+This box is made to run the full setup, but if you are not sure or don't want to wait, we recommend to start with the light setup. You can switch to the full setup at any time.
+
+## Reference Shopping List
+* [GIGABYTE GB-BLCE-4105 BRIX](https://www.computeruniverse.net/en/gigabyte-gb-blce-4105-brix): 148,99 €
+  * Alternative: [ODROID-H2](https://www.hardkernel.com/shop/odroid-h2/), same platform but sold out since ages and needs separate case+power
```

Comment:
Maybe list https://www.seeedstudio.com/ODYSSEY-X86J4105864-p-4447.html as alternative since that one is sold out

Reply:
Done